js/console.go

```diff
@@ -75,22 +80,40 @@ func (c console) log(level logrus.Level, msgobj goja.Value, args ...goja.Value)
 	}
 }
 
-func (c console) Log(msg goja.Value, args ...goja.Value) {
-	c.Info(msg, args...)
+func (c console) Log(args ...goja.Value) {
+	c.Info(args...)
+}
+
+func (c console) Debug(args ...goja.Value) {
+	c.log(logrus.DebugLevel, args...)
 }
 
-func (c console) Debug(msg goja.Value, args ...goja.Value) {
-	c.log(logrus.DebugLevel, msg, args...)
+func (c console) Info(args ...goja.Value) {
+	c.log(logrus.InfoLevel, args...)
 }
 
-func (c console) Info(msg goja.Value, args ...goja.Value) {
-	c.log(logrus.InfoLevel, msg, args...)
+func (c console) Warn(args ...goja.Value) {
+	c.log(logrus.WarnLevel, args...)
 }
 
-func (c console) Warn(msg goja.Value, args ...goja.Value) {
-	c.log(logrus.WarnLevel, msg, args...)
+func (c console) Error(args ...goja.Value) {
+	c.log(logrus.ErrorLevel, args...)
 }
 
-func (c console) Error(msg goja.Value, args ...goja.Value) {
-	c.log(logrus.ErrorLevel, msg, args...)
+func (c console) valueString(v goja.Value) string {
+	exptype := v.ExportType()
+	if exptype == nil {
+		return v.String()
+	}
+
+	kind := exptype.Kind()
+	if kind != reflect.Map && kind != reflect.Slice {
+		return v.String()
+	}
+
+	b, err := json.Marshal(v.Export())
+	if err != nil {
```

Comment:
Should we do a warning log here, maybe, or it's too much? :thinking:

Reply:
Do you have any suggestions about the warning?

Reply:
Nothing special, my concern was that if we do not log anything it hard to realize that something wrong with marshaling, but looking into it right now it seems like not the big deal